.buildkite/dagster-buildkite/tox.ini

```diff
@@ -6,7 +6,7 @@ skipsdist = true
 usedevelop = true
 setenv =
   VIRTUALENV_PIP=21.3.1
-passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE SNOWFLAKE_ACCOUNT SNOWFLAKE_USER SNOWFLAKE_PASSWORD
+passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE* SNOWFLAKE_ACCOUNT SNOWFLAKE_USER SNOWFLAKE_PASSWORD
```

Comment:
Alternatively, we could allow only the envvars needed for the test collector to properly label things:

https://buildkite.com/docs/test-analytics/ci-environments#buildkite

Instead of every BUILDKITE prefixed envvar:

https://buildkite.com/docs/pipelines/environment-variables

I think the complete list is pretty innocuous though.